service/src/main/java/uk/gov/hmcts/reform/fpl/service/RepresentativeService.java

```diff
@@ -190,7 +190,7 @@ private void associatedRepresentativesWithParties(CaseData caseData) {
             .map(Element::getValue)
             .map(Representable::getRepresentedBy)
             .forEach(List::clear);
-        caseData.getAllOthers().forEach(o -> o.getRepresentedBy().clear());
+        caseData.getAllOthers().forEach(o -> o.getValue().getRepresentedBy().clear());
```

Comment:
Let's be more specific with naming here, can we go `other`?

Reply:
existing code in another service, so I am not going to refactor. I only touched becaause getAllOthers returns List<Element<Other>> instead of List<Other> now